core/src/epicli/data/common/ansible/playbooks/roles/node_exporter/tasks/main.yml

```
@@ -1,83 +1,113 @@
 ---
-- name: Create node_exporter system group
-  group:
-    name: node_exporter
-    system: true
-    state: present
+- name: Install Node Exporter as System Service
+  block:
+    - name: Create node_exporter system group
+      group:
+        name: node_exporter
+        system: true
+        state: present
 
-- name: Create node_exporter system user
-  user:
-    name: node_exporter
-    system: true
-    shell: "/usr/sbin/nologin"
-    group: node_exporter
-    createhome: false
+    - name: Create node_exporter system user
+      user:
+        name: node_exporter
+        system: true
+        shell: "/usr/sbin/nologin"
+        group: node_exporter
+        createhome: false
 
-- name: Set Node Exporter file name to install
-  set_fact:
-    exporter_file_name: "node_exporter-{{ exporter.version }}.linux-amd64.tar.gz"
+    - name: Set Node Exporter file name to install
+      set_fact:
+        exporter_file_name: "node_exporter-{{ exporter.version }}.linux-amd64.tar.gz"
 
-- name: Download Node Exporter binaries
-  include_role:
-    name: download
-    tasks_from: download_file
-  vars:
-    file_name: "{{ exporter_file_name }}"
+    - name: Download Node Exporter binaries
+      include_role:
+        name: download
+        tasks_from: download_file
+      vars:
+        file_name: "{{ exporter_file_name }}"
 
-- name: Create /opt/node_exporter directories
-  become: true
-  file:
-    path: "{{ item }}"
-    recurse: true
-    owner: root
-    group: "node_exporter"
-    mode: 0750
-    state: directory
-  with_items:
-  - /opt/node_exporter
+    - name: Create /opt/node_exporter directories
+      become: true
+      file:
+        path: "{{ item }}"
+        recurse: true
+        owner: root
+        group: "node_exporter"
+        mode: 0750
```

How about going human readable? :thinking: 
```suggestion
        mode: u=rwx,g=rx,o=
```

---

Done